app/views/categories/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="top-nav py-4">
+<div class="container">
+  <div class="row">
+  <div class="col">
+   <button class="navbar-toggler ms-auto text-white" type="button" data-mdb-toggle="collapse"
+      data-mdb-target="#navbarToggleExternalContent3" aria-controls="navbarToggleExternalContent3"
+      aria-expanded="true" aria-label="Toggle navigation">
+      <i class="fas fa-bars"></i>
+    </button>
+    </div>
+    <div class="col">
+    <h1 class="text-white">TRANSACTIONS</h1>
+    </div>
+    <div class="col justify-content-end d-flex">
+    <div><%= button_to "Sign out", destroy_user_session_path, method: :delete, class: 'btn btn-danger' %></div>
+    </div>
+  </div>
+  </div>
+  <div class="container d-flex justify-content-end gap-3 mt-3 mb-3">
+   
+    <button>Most Recent</button>
+    <button>Most Ancient</button>
+  </div>
+  </div>
+  <div id="categories" class="container bg-light">
+    <% @categories.each do |category| %>
+      <%= link_to "/categories/#{category.id}", class: "text-decoration-none" do %>
+        <div class="container bg-white mt-4 shadow-lg p-3 mb-5 rounded">
+          <div class="row">
+            <div id="img-div" class="col"><img src=<%= category.icon%> alt=<%= category.name%> /></div>
+            <div class="col">
+              <p class = "fw-bold text-secondary fs-5"><%= category.name %><p>
+              <p class = "fw-light text-secondary"><%= category.created_at.to_date.strftime('%Y %b %d') %></p>
+            </div>
+            <div class="col d-flex justify-content-end">
```

Comment:
- [ ] Let's style this card accordingly to the given design to show professional work.

<img width="1438" alt="Screenshot 2022-12-03 at 11 40 14 PM" src="https://user-images.githubusercontent.com/40503704/205465089-d9aa4f2d-5c6b-4922-9092-1174bb426837.png">